spec/forms/openid_connect_authorize_form_spec.rb

```diff
@@ -707,37 +707,4 @@
       end
     end
   end
-
-  describe '#biometric_comparison_requested?' do
```

Comment:
Did you mean to remove this method from the form object? I do not see that change in this PR

Reply:
yes i did! thanks, will update

Reply:
fixed in https://github.com/18F/identity-idp/pull/11007/commits/dce21570ba4cd8f930ed51e029d6d2aac132101d